CHANGELOG.md

```diff
@@ -3,14 +3,6 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.0.1] - TBD
```

Comment:
shouldn't this be `1.0.0` ?

Reply:
We will release it as 1.0.0 once we know that it all works as expected.